qa/workunits/cephadm/nvme_initiator.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -ex
+
+sudo modprobe nvme-fabrics
+sudo modprobe nvme-tcp
+
+GATEWAY=$(cat /etc/ceph/nvmeof.cfg |grep 'gateway_addr' | awk -F'[, ]' '{print $3}')
+IP=$(cat /etc/ceph/nvmeof.cfg |grep 'ip_address' | awk -F'[, ]' '{print $3}')
+HOSTNAME=$(hostname)
+
+echo -e "<---- exec.client.0---- HOST/IP -- $HOSTNAME/$IP ---->"
```

Comment:
I believe the echo was added for debugging purposes only, right?

Reply:
yeah, I'll remove it since we have `-ex` option enabled for this script anyway!

Reply:
> yeah, I'll remove it since we have `-ex` option enabled for this script anyway!

While at it, I would suggest getting rid of the clearly useless listing commands, such as `ceph config get` for an option right after setting it or `rbd ls` right after creating the image in the newly created pool.

Reply:
agree